cirq/sim/mps_simulator_test.py

```diff
@@ -0,0 +1,196 @@
+import math
+
+import numpy as np
+import pytest
+import sympy
+
+import cirq
+
+
+def assert_same_output_as_dense(circuit, qubit_order, initial_state=0):
+    mps_simulator = cirq.MPSSimulator()
+    ref_simulator = cirq.Simulator()
+
+    actual = mps_simulator.simulate(circuit, qubit_order=qubit_order, initial_state=initial_state)
+    expected = ref_simulator.simulate(circuit, qubit_order=qubit_order, initial_state=initial_state)
+    np.testing.assert_almost_equal(actual.final_state.to_numpy(), expected.final_state_vector)
+    assert len(actual.measurements) == 0
+
+
+def test_various_gates():
+    gate_cls = [cirq.I, cirq.H, cirq.X, cirq.Y, cirq.Z]
```

Comment:
Please add `cirq.T` to this set to make it universal.

Reply:
Done.